frontend/src/layout/scenes/components/SceneBreadcrumbs.tsx

```diff
@@ -11,27 +11,34 @@ import { Breadcrumb } from '~/types'
 interface SceneBreadcrumbBackButtonProps {
     forceBackTo?: Breadcrumb
     className?: string
+    iconOnly?: boolean
```

Comment:
GENERAL

I am muuuuuch bigger fan of this than v2. This to me makes much more sense